acl.go

```diff
@@ -42,30 +48,34 @@ func (odbi *ovnDBImp) getACLUUIDByRow(lsw, table string, row OVNRow) (string, er
 					if as, ok := acls.(libovsdb.OvsSet); ok {
 						for _, a := range as.GoSet {
 							if va, ok := a.(libovsdb.UUID); ok {
+								cacheACL, ok := odbi.cache[tableACL][va.GoUUID]
+								if !ok {
+									return "", ErrorNotFound
+								}
 								for field, value := range row {
 									switch field {
 									case "action":
-										if odbi.cache[tableACL][va.GoUUID].Fields["action"].(string) != value {
+										if cacheACL.Fields["action"].(string) != value {
```

Comment:
in future, maybe have head file to define the table structure, and here can be simplified to (TABLE field name), (TABLE field type).

thoughts?

Reply:
i'm prefer to try to utilize hashicorp/mapstructure as row have map[string]interface{} format we can automatic unmrashal this row to struct, for types like OvsSet and other specific we can use custom decoder...
So huge part of code can be replaced.

Reply:
same idea with me,  could we have issue ticket to track it and discuss.

Reply:
https://github.com/eBay/go-ovn/issues/29 needs more generic name..